crates/router/src/configs/defaults.rs

```diff
@@ -4465,14 +4465,244 @@ impl Default for super::settings::RequiredFields {
+                                                "payment_method_data.bank_redirect.giropay.bank_account_bic".to_string(),
+                                                RequiredFieldInfo {
+                                                    required_field: "payment_method_data.bank_redirect.giropay.bank_account_bic".to_string(),
+                                                    display_name: "bank_account_bic".to_string(),
+                                                    field_type: enums::FieldType::Text,
+                                                    value: None,
+                                                }
+                                            ),
+                                            (
+                                                "payment_method_data.bank_redirect.giropay.bank_account_iban".to_string(),
+                                                RequiredFieldInfo {
+                                                    required_field: "payment_method_data.bank_redirect.giropay.bank_account_iban".to_string(),
+                                                    display_name: "bank_account_iban".to_string(),
+                                                    field_type: enums::FieldType::Text,
+                                                    value: None,
+                                                }
+                                            ),
+                                            (
+                                                "payment_method_data.bank_redirect.giropay.country".to_string(),
+                                                RequiredFieldInfo {
+                                                    required_field: "payment_method_data.bank_redirect.giropay.country".to_string(),
+                                                    display_name: "bank_account_iban".to_string(),
+                                                    field_type: enums::FieldType::UserCountry {
+                                                        options: vec![
+                                                                "DE".to_string(),
+                                                        ]},
+                                                    value: None,
+                                                }
+                                            )
+                                        ]),
+                                        common: HashMap::new(),
+                                    }
+                                ),
+                                (
+                                    enums::Connector::Adyen,
+                                    RequiredFieldFinal {
+                                        mandate: HashMap::new(),
+                                        non_mandate: HashMap::new(),
+                                        common: HashMap::new(),
+                                    }
+                                ),
```

Comment:
if Adyen supports mandates, could you please confirm if this config is present in toml file?
```
[mandates.supported_payment_methods]
bank_redirect.giropay
```
Could you please check for other connectors?

Reply:
Adyen supports mandates. I will add it to the config files

Reply:
Also, make sure you update the env files which is located at https://github.com/juspay/hyperswitch/tree/main/config/deployments